packages/gatsby-module-loader/src/patch.js

```diff
@@ -1,80 +0,0 @@
-patch()
-
-function patch() {
```

Comment:
This seemed to be solely for better error handling, but v2 handles errors better so I don't know if it's still needed. I wasn't sure how to update it to be v2 compatible tho

Reply:
Yeah that's what I remember is it was just to add error reporting on failed loads